web/src/components/DandisetsPage.vue

```diff
@@ -39,33 +40,53 @@
           </v-list-item>
         </v-list>
       </v-menu>
-      <div class="mx-6">
-        Sort By:
-      </div>
-      <v-chip-group
-        :model-value="sortOption"
-        selected-class="white text-light-blue bg-white"
-        dark
-        mandatory
-        style="min-width: 25%"
+      <DandisetSearchField class="flex-grow-1 mr-2" />
+      <v-btn-group
+        variant="flat"
+        density="comfortable"
+        divided
+        class="btn-group--sort-options"
       >
-        <v-chip
-          v-for="(option, i) in sortingOptions"
-          :key="option.name"
-          @click="changeSort(i)"
+        <v-btn class="btn--sort-option">
+          <v-icon>mdi-sort-variant</v-icon>
+          <v-tooltip
+            location="top"
+            activator="parent"
+          >
+            <span>Sorting Options</span>
+          </v-tooltip>
+          <v-menu activator="parent">
+            <v-list>
+              <v-list-item
+                v-for="(option, i) in sortingOptions"
+                :key="option.name"
+                :value="option"
+                :append-icon="option.name === sortingOptions[sortOption].name ? 'mdi-check' : ''"
```

Comment:
Now that we're using `v-list` here, you should be able to use Vuetify's built in data binding capabilities (i.e., set `v-model="sortingOptions"` and let Vuetify set `sortOption` implicitly) instead of manually catching the `click` event and updating `sortOption` explicitly in `changeSort()`.

Reply:
See b2e7fee for my approach here. I couldn't seem to get `v-model` working with `v-list`, so it seemed like I had to wrap this bit in ` v-item-group`. It removes logic from the template and let me delete the `changeSort` function from the script, so there's definitely upside to using the approach, although it's still not obvious to me if this is _correct_ per se, or if I was missing something from the `VList` documentation.